tests/manage/z_cluster/test_get_requests_by_kube_probe.py

```diff
@@ -0,0 +1,30 @@
+import logging
+import re
+
+from ocs_ci.ocs.resources.pod import get_pod_logs
+from ocs_ci.ocs.utils import get_pod_name_by_pattern
+from ocs_ci.framework.testlib import (
+    tier2,
+    bugzilla,
+    skipif_ocs_version,
+)
+
+log = logging.getLogger(__name__)
+
+
```

Comment:
From the BZ, it looks like the issue was seeing after the upgrades. The BZ was verified checking the dashboard status after ocp and odf upgrades. Shouldn't the test be automated and run the same way?

Reply:
@PrasadDesala We test it in test_validation_ui.
https://github.com/red-hat-storage/ocs-ci/blob/master/tests/ui/test_validation_ui.py#L35